### ISSUE TITLE
ImageSourcePropType is deprecated in RN 0.58.3

### ISSUE THREAD

**Issue by ashubits11** [merryjs/photo-viewer#71]
I was updating my application to latest version of react native, After update i am getting error ImageSourcePropType not found.
In your package you have use 
const ImageSourcePropType = require("react-native/Libraries/Image/ImageSourcePropType");
Which is no more available.
Please check in following url:-
https://github.com/facebook/react-native/tree/master/Libraries/Image

**Comment by nwaughachukwuma** [comment]
Same here. Please are there plans to update this library to React Native 0.58.x?

**Comment by lioialessandro** [comment]
Hi, I've made a pull request to fix this issue (#70 ). You can use this version by appending `#refs/pull/70/head` to the @merryjs/photoviewer dependencies in package.json so it looks like this
```
...
"@merryjs/photo-viewer": "merryjs/photo-viewer#refs/pull/70/head"
...
```
